test/cri-containerd/policy_test.go

```diff
@@ -772,6 +773,8 @@ func Test_SecurityPolicyEnv_Annotation(t *testing.T) {
 				defer removeContainer(t, client, ctx, containerID)
 				defer stopContainer(t, client, ctx, containerID)
 
+				// add a sleep for stdio to kick in
+				time.Sleep(time.Second)
```

Comment:
We've done this in the past for a couple things but I'd like to try and not rely on sleeps, as even some timer where I think would be perfectly fine always seems to go awry in tests on different hardware.. containerd handles this in their tests by having a construct to wait for the container to exit before reading the file https://github.com/containerd/containerd/blob/main/integration/container_log_test.go#L62-L75